keanu-project/src/test/java/io/improbable/keanu/e2e/regression/LinearLassoRegressionTest.java

```diff
@@ -55,7 +55,7 @@ public void findsExpectedParamsForManyWeights() {
 
         RegressionModel linearRegressionModel = RegressionModel.withTrainingData(data.xTrain, data.yTrain)
             .withRegularization(RegressionRegularization.LASSO)
-            .withPriorOnIntercept(0, 40)
+            .withPriorOnIntercept(0, 20)
```

Comment:
Think you fixed this on another PR?

Reply:
Yup, it was just to get it building locally. I'll pull develop in.